be1-go/channel/coin/uint53/mod.go

```diff
@@ -0,0 +1,37 @@
+/*
+  Operations on 53-bits non-negative integers.
+
+  These values can be handled without loss of precision in IEEE754 64-bit
+  floating numbers, and thus in JSON.
+*/
```

Comment:
You must use `//` and the comment should start with "Package uint53 ...".
See https://go.dev/doc/comment. Also be sure comment lines are wrapped in 80 chars.